### DISCUSSION TITLE
Request for updating Zephyrus g14 2022 GA402 Hackintosh

### DISCUSSION THREAD

**Discussion by Mihrock1** [PIut02/ROG-Zephyrus-G14-GA401IV-Hackintosh#25] [General]
I have been following the development of Hackintosh for Zephyrus G14 2022 model : https://github.com/b00t0x/ROG-Zephyrus-G14-GA402-Hackintosh
Unfortunately the project has not been updated in a long time (more than 4 months). Seeing the progress of Hackintosh for 2021 and 2020 model here, I wonder if its possible for you to take up the development of 2022 model further. The biggest issue with this model is the lack of working function keys and also any adjustment of hardware display brightness. 
NootedRed has a while back, talked about working on for the support of iGPU that comes with 2022 model. Once that is out, a working iGPU can also fix the bad battery life on the 2022 model where currently only the discrete AMD GPU is working.
PS:- I can actively test for you, in case you dont have the the 2022 model.

**Comment by PIut02** [comment]
I think it will be a long time to wait for NootedRed to support 2022 models, and I have no idea about the issues, so no plans for now.
If NootedRed supports the 2022 model, then I will consider buying one and creating a new repository for it.
